src/apiserver/api_server_impl.cc

```diff
@@ -83,6 +83,67 @@ void APIServerImpl::Process(google::protobuf::RpcController* cntl_base, const Ht
     cntl->response_attachment().append(writer.GetString());
 }
 
+class ExecContext {
+ public:
+    bool is_online = false;
+    bool is_sync = true;
```

Comment:
it's better to declare with struct here. refer https://google.github.io/styleguide/cppguide.html#Variable_Names.

Reply:
done